dataplane/worker.h

```diff
@@ -51,6 +51,11 @@ class tStack
 		mbufsCount = 0;
 	}
 
+	inline void copy_from(tStack& other)
```

Comment:
```suggestion
	void copy_from(tStack& other)
```
`inline` is redundant.

> A function defined entirely inside a class/struct/union definition, whether it's a member function or a non-member friend function, is implicitly an inline function

[link](https://en.cppreference.com/w/cpp/language/inline)